api/ImageBitmap.json

```diff
@@ -55,7 +55,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ImageBitmap/height",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
This is `50`. Please change for Chrome and Chrome_Android as well.

Reply:
Added. Also changed Opera to "37".